Samples.xcodeproj/project.pbxproj

```diff
@@ -177,14 +179,12 @@
 			dstPath = "";
 			dstSubfolderSpec = 7;
 			files = (
+				1C0C1C5729D76768005C8B24 /* RealisticLightingAndShadowsView.swift in Copy Source Code Files */,
 				883C121729C914E100062FF9 /* DownloadPreplannedMapAreaView.MapPicker.swift in Copy Source Code Files */,
 				883C121829C914E100062FF9 /* DownloadPreplannedMapAreaView.Model.swift in Copy Source Code Files */,
 				883C121929C914E100062FF9 /* DownloadPreplannedMapAreaView.swift in Copy Source Code Files */,
 				1C0C1C3D29D34DDD005C8B24 /* ChangeViewpointView.swift in Copy Source Code Files */,
 				1C42E04A29D239D2004FC4BE /* ShowPopupView.swift in Copy Source Code Files */,
-				108EC04329D25D6D000F35D0 /* ChangeMapViewBackgroundView.swift in Copy Source Code Files */,
-				108EC04429D25D6D000F35D0 /* ChangeMapViewBackgroundView.SettingsView.swift in Copy Source Code Files */,
-				108EC04529D25D6D000F35D0 /* ChangeMapViewBackgroundView.Model.swift in Copy Source Code Files */,
```

Comment:
This doesn't look correct. Did you pull from the latest `v.next` before you created your branch? 🤔

Reply:
Closing this PR in favor of [this](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/151) one. These files were accidentally removed in this PR and I'm not sure how to add them back because I'm not able to rebase onto `v.next` since the branch is up-to-date.

Reply:
No problem. Next time if you have problem with xcodeproj we can have a quick chat and try to fix them in place.